apps/web/src/pages/templates/editor/TemplateEditorPage.tsx

```diff
@@ -10,12 +10,22 @@ import { useEnvController } from '../../../store/use-env-controller';
 import WorkflowEditorPage from '../workflow/WorkflowEditorPage';
 import { TemplateEditor } from '../../../components/templates/TemplateEditor';
 import { TemplateSettings } from '../../../components/templates/TemplateSettings';
+import { TemplatePageHeader } from '../../../components/templates/TemplatePageHeader';
+
+export enum ActivePageEnum {
+  Settings = 'Settings',
+  Workflow = 'Workflow',
+  sms = 'Sms',
+  email = 'Email',
+  in_app = 'in_app',
+  TriggerSnippet = 'TriggerSnippet',
```

Comment:
```suggestion
  SETTINGS = 'Settings',
  WORKFLOW = 'Workflow',
  SMS = 'Sms',
  EMAIL = 'Email',
  IN_APP = 'in_app',
  TRIGGER_SNIPPET = 'TriggerSnippet',
```

a small suggestion to have a similar casing for properties? We use Pascal, camel and snake case all in the same place :P